Resources/Prototypes/_DV/Entities/Objects/Specific/Security/security.yml

```diff
@@ -68,10 +68,3 @@
   id: HoloprojectorSecurityRecharging
   suffix: Recharging, Do Not Map
   categories: [ HideSpawnMenu ]
```

Comment:
Maybe just remove this entity, swap out the BorgModuleSecurityPatrol for a normal one (that's the only place this is used). Would reduce clutter is all

Reply:
Seconded, ditch this and only use HoloprojectorSecurity, it's all in _DV anyway

Reply:
Done.